src/plugins/unified_histogram/public/chart/histogram.tsx

```diff
@@ -116,17 +116,15 @@ export function Histogram({
     timeField: dataView.timeFieldName,
   });
   const chartRef = useRef<HTMLDivElement | null>(null);
-  const { height: containerHeight, width: containerWidth } = useResizeObserver(chartRef.current);
   const { attributes } = visContext;
 
   useEffect(() => {
     if (attributes.visualizationType === 'lnsMetric') {
-      const size = containerHeight < containerWidth ? containerHeight : containerWidth;
-      setChartSize(`${size}px`);
+      setChartSize('90%');
```

Comment:
This approach works, but since we don't need the chart dimensions anymore, we could remove this effect entirely and just choose between `90%` and `100%` in the `.lnsExpressionRenderer` CSS like we do for `box-shadow`. I feel like it would simplify things a bit. I pushed a quick commit with the change here to show what I mean if you're interested: https://github.com/davismcphee/kibana/commit/fd047e10a7df66647b1e324db537e69ae863596a?diff=split&w=1.

Reply:
Good point, I did it very quickly, done!